docs/2.0/upgrade.md

```diff
@@ -28,39 +35,6 @@ Avo.configure do |config|
   end
 ```
 
-## Upgrade from 2.27 to 2.28
```

Comment:
This should still be present IMHO. Maybe someone doesn't upgrade tot he latest version, or they need the historic data.

Reply:
This upgrade guide is what we write for first approach, when both attachments and field level authorization was required. Before the deploy we did an commit where we loose the authorization, than only 1 of the mentioned above where needed. That loosing commit removed the breaking change and this never was truly a breaking change